### PR TITLE
Updated the questions for interview with Jeremy

### PR DESCRIPTION
**Pull request by VidyaRajesh2019**
Hi Jeremy,

Completed interview lab part 1 and part 2. Please review them and provide your feedback.

Thanks,
Vidya.

### CODE REVIEW

File: interview [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Q1: So you're finishing up your time in this CSCC program. Why did you decide to embark on this journey?
+I want to become a full stack developer as industry is going towards that. Also, I want to learn Java programming which is in great demand.
+
+Q2: I love to travel. Tell me a bit about your favorite travel destination so far. What's on your travel bucket list?
+My favorite travel destination was Bahamas cruise.
+I want to goto Alaska cruise, Europe tour and Banff.
```
Comment (jschmersal-cscc) [inline interview:6]: Great places on your list!  I'm going on a cruise this summer (not Alaska) with our family and the in-laws.  Should be a great time!  I totally want to do Alaska/Banff.  I really want to see some of that wonderful nature!

File: interview [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+Q3: Name your favorite member of The Beatles, and your favorite Beatles song. You do know at least one Beatles song, right? If you are a Beatles fan, feel free to give me your top 5 or so.
+Hey Jude, Yesterday, Don't let me down, Blackbird and something are some of my favorites Beatles song.
```
Comment (jschmersal-cscc) [inline interview:9]: That's a great list.  They're all up there on my list too!

File: interview [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+Q4: Do you have any pets? Species/Name/Details please. Murray wants to know... (he's not as smart as that picture makes him seem).
+I don't have any pets. But my son loves pets. Planning to have it in future.
```
Comment (jschmersal-cscc) [inline interview:12]: Murray would approve of your son's pet plans :)

**Review by jschmersal-cscc — APPROVED** [review]
Good job! You are finished with this part of the lab. If you haven't yet submitted your assignment for grading, please include a screenshot of this pull request page with it. If you have, don't worry!

File: interview (hunk 1 above)
Comment (VidyaRajesh2019) [reply interview:6]: wow nice...Have a nice and safe trip.

**Pull request closed without merging**